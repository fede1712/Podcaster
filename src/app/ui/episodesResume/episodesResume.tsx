import React from "react";

export const EpisodesResume = ({ data }: { data: { resultCount: number } }) => {
  return <p className="border shadow-md p-4 font-bold">Episodes: {data.resultCount}</p>;
};
