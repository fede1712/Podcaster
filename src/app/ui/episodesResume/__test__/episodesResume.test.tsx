import React from "react";
import { render, screen } from "@testing-library/react";
import { EpisodesResume } from "../episodesResume";

test("Renders episode resume correctly", () => {
  render(<EpisodesResume data={{ resultCount: 10 }} />);

  const episodesResumeElement = screen.queryByText("Episodes: 10");
  expect(episodesResumeElement).not.toBeNull();

  const { asFragment } = render(<EpisodesResume data={{ resultCount: 10 }} />);

  expect(asFragment()).toMatchSnapshot();
});
