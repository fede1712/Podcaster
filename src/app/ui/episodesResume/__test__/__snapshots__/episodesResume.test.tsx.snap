// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders episode resume correctly 1`] = `
<DocumentFragment>
  <p
    class="border shadow-md p-4 font-bold"
  >
    Episodes: 10
  </p>
</DocumentFragment>
`;
